src/PointwiseFunctions/Hydro/Python/ComovingMagneticField.hpp

```diff
@@ -0,0 +1,13 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <pybind11/pybind11.h>
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+
+namespace py_bindings {
+// NOLINTNEXTLINE(google-runtime-references)
+void bind_comovingMF(pybind11::module& m);
```

Comment:
`bind_comoving_magnetic_field` (this violates naming convention and MF is a bit of an unfortunate abbreviation that I don't think we should have in a public code 🙃 )